spec/customer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'customer.rb'
+
+describe Customer do
+
+  it 'should respond to view menu' do
+    c1 = Customer.new 12345
+    expect(c1).to respond_to :menu
+  end
+
+  it 'should respond to place order' do
+    c1 = Customer.new 12345
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).